Misc/NEWS.d/next/Build/2024-08-01-10-51-02.gh-issue-122571.fpA1xD.rst

```diff
@@ -0,0 +1,4 @@
+The PY_BUILTIN_HASHLIB_HASHES variable was defined twice in configure.ac,
```

Comment:
I'm not really sure whether a NEWS is needed, but if it is, it should be clearer (it's as if I was reading [X => Y => Z] + bad test).

Reply:
Personally I'd be in favor of no news, but I wanted to get CI green. If you can mark this PR as no NEWS entry needed, I'll remove the entry from the PR

Reply:
I'll do it (because the NEWS is more confusing that helping IMO...)

Reply:
Great, I force-pushed to remove the latest commit (NEWS entry). This PR is now a single commit that fixes configure.ac. Thank you.